### DISCUSSION TITLE
How to fix my npm install doesn't work?

### DISCUSSION THREAD

**Discussion by Saidur07** [opensumi/core#1604] [Q&A] [answered]
How to fix my npm install doesn't work? It show's some errors. I have to run npm i -f. How to get rid of that?

**Comment by bytemain** [comment]
Please add some specific information(such as a screenshot) about the error, so we can solve the problem for you

**Comment by mahfuz47** [comment, marked as answer]
you can use npm install --force or can follow the documentation https://docs.npmjs.com/cli/v6/commands/npm-install

**Comment by erha19** [comment]
@programmer-saidur If you are using NPM@7+, try to add `--legacy-peer-deps` ignores the new requirement, at the risk of introducing breaking changes
